op-e2e/system/da/throughput_test.go

```diff
@@ -0,0 +1,73 @@
+package da
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	op_e2e "github.com/ethereum-optimism/optimism/op-e2e"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/geth"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/transactions"
+	"github.com/ethereum-optimism/optimism/op-e2e/system/e2esys"
+	"github.com/stretchr/testify/require"
+)
+
+// TestBatcherThroughput tests the throughput of the batcher by creating a large amount of L2 data
+// before starting the batcher. This causes a backlog which must be cleared quickly by the batcher.
+func TestBatcherThroughput(t *testing.T) {
```

Comment:
Not sure about adding this test, yet. So far been unable to show any difference in throughput with the changes from this PR.